Dockerfile

```diff
@@ -17,7 +17,7 @@ COPY --from=frontend /app/assets ./assets
 # copy generated index page
 COPY --from=frontend /app/templates ./templates
 # copy package.json & package-lock.json
-COPY package*.json .
+COPY package*.json ./
```

Comment:
weird

Reply:
Not sure why we need to copy more than one packages json file , it was in the original Dockerfile , but  when are copied multiple files with COPY command , the path needs end with "/" at end to work.